resources/views/about/roles/community-organizations-consultation-participants.blade.php

```diff
@@ -0,0 +1,107 @@
+<x-app-layout page-width="wide" header-class="full header--participants stack">
+    <x-slot name="title">{{ __('Consultation Participants') }}</x-slot>
+    <x-slot name="header">
+        <div class="center center:wide stack pt-4 pb-12">
+            <ol class="breadcrumbs" role="list">
+                <li><a href="{{ localized_route('welcome') }}">{{ __('About the Accessibility Exchange') }}</a></li>
+                @if (request()->localizedRouteIs('about.individual-consultation-participants'))
+                    <li><a
+                            href="{{ localized_route('about.for-individuals') }}">{{ __('How this works for individuals') }}</a>
+                    </li>
+                @elseif(request()->localizedRouteIs('about.organization-consultation-participants'))
+                    <li><a
+                            href="{{ localized_route('about.for-community-organizations') }}">{{ __('How this works for Community Organizations') }}</a>
+                    </li>
+                @endif
```

Comment:
I guess this conditional isn't needed because you've split the routes now. I assume the same is true for the original view too.

Reply:
I guess you might not need to split this into a new file, if you can use a conditional for the steps. But if you feel it is better to keep them separate that is fine too.